src/app/lib/utilities/variantHandler/index.js

```diff
@@ -4,6 +4,7 @@ export const servicesWithVariants = {
   // the first element in the array is the default variant
   serbian: ['lat', 'cyr'],
   ukchina: ['simp', 'trad'],
+  uzbek: ['default', 'cyr', 'lat'],
```

Comment:
I think we can actually remove 'default' from here and in turn the Uzbek service config as well, as this line here: https://github.com/bbc/simorgh/blob/467ab3f2037cee2438b6a3de43dcad2672222973/src/app/lib/utilities/variantHandler/index.js#L30 will default to `cyr` anyway.

This means we won't actually need the concept of `default` for Uzbek, relying just on `cyr` being the default which is the case.

Reply:
Hmm, removing the above causes the Homepage to redirect from `/` to `/cyr`

Reply:
We could add a temporary check for uzbek here: https://github.com/bbc/simorgh/blob/467ab3f2037cee2438b6a3de43dcad2672222973/src/app/legacy/containers/PageHandlers/withVariant/getVariantRedirectUrl/index.js#L25

so something like:
```
const isUrlVariantRedirect = !variant && defaultVariant !== 'default' && service !== 'uzbek'; 
```

This would be temporary until the Homepage work is done to default to `/cyr`, after which that extra condition could be removed.

Reply:
I see some of the tests are failing regarding looking for the `default` config, forgot about that `getConfig` function.

Perhaps it is better to leave it as you had it, as it worked well. We can always look again at removing the `default` stuff in the future if we can, but its not a big deal.

Reply:
Or we may just need to revert https://github.com/bbc/simorgh/pull/11685/commits/0fae4c505f88076df305fab9e73a2fd1b07cd1dc and see if that works. Just so long as the legacy STY assets continue to work with the correct config/theme.